src/main/java/ch/jalu/configme/properties/types/StringType.java

```diff
@@ -0,0 +1,37 @@
+package ch.jalu.configme.properties.types;
+
+import ch.jalu.configme.properties.convertresult.ConvertErrorRecorder;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Locale;
+
+public class StringType extends PropertyAndLeafType<String> {
+
+    public static final StringType STRING = new StringType();
+
+    public static final StringType STRING_LOWER_CASE = new StringType() {
+        @Override
+        protected @NotNull String transformToString(@NotNull Object object) {
+            return object.toString().toLowerCase(Locale.ROOT);
+        }
+    };
+
+    protected StringType() {
+        super(String.class);
+    }
+
+    @Override
+    public @Nullable String convert(@Nullable Object object, @NotNull ConvertErrorRecorder errorRecorder) {
+        return object == null ? null : transformToString(object);
+    }
+
+    @Override
+    public @Nullable Object toExportValue(@Nullable String value) {
```

Comment:
can be `@NotNull String toExportValue` , though I think I already changed that in the property builder draft branch